### ISSUE TITLE
Add an RSS feed for blog posts

### ISSUE THREAD

**Issue by feedm3** [satellytes/satellytes.com#99]
We want to offer an RSS feed for our blog posts so that the users can add our blog to their favorite RSS reader.

There is a tutorial on Gatsbyjs.com to do this:
- https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-an-rss-feed/
- https://www.gatsbyjs.com/plugins/gatsby-plugin-feed/

We also need to clarify:
1. Do wee need an RSS button on every blog post? Or only on the blog overview page?
2. Do wee need to do something else to make it searchable within famous RSS readers? I'm thinking of Feedly, but we can do some small research to check if there are other famous ones.

**Comment by georgiee** [comment]
I think it's enough to offer the rss url as some small link in the footer. People can then add it to their reader like feedly. I bet they show/index any content people added. I could not find any explicit "index" where we could register. Let's start with the url and then figure out the next steps.

**Comment by georgiee** [comment]
I can spend some time on this today, mainly following the tutorial and establishing the rss url without any changes in the UI for the first draft.

**Comment by georgiee** [comment]
I will give back this issue to the queue. It's not working out of the box with the plugin `gatsby-plugin-feed`. It requires a full build which stretches the time to debug and incrementally improve the config. It's only relevant for the blog where we miss the slug, so I tried a custom serializer + graphql as described on the plugin page (https://www.gatsbyjs.com/plugins/gatsby-plugin-feed/). This is very close to the default implementation.

I got null values for the path, tinkered around a little bit but currently I don't have the time to dig much deeper. Maybe someone else can pick up this topic. Thanks.
